src/Accounts/Accounts/Common/EnvironmentCompleterAttribute.cs

```diff
@@ -28,13 +31,16 @@ public static string[] GetEnvironments()
             return profileClient.ListEnvironments(null).Select(x => x.Name).ToArray();
         }
 
-        private static ScriptBlock CreateScriptBlock()
+        /// <summary>
+        /// Implementations CompleteArgument function of the <see cref="IArgumentCompleter"/>.
+        /// </summary>
+        public IEnumerable<CompletionResult> CompleteArgument(string commandName, string parameterName, string wordToComplete, CommandAst commandAst, IDictionary fakeBoundParameters)
```

Comment:
As we talked offline, the test cases  should be updated to test the equality of the results of `GetEnvironments()` and `CompleteArgument()`.

Reply:
Done